### PR TITLE
testcases for Developer Options

### PR DESCRIPTION
**Pull request by kbo001**
## Summary
Automated the tests developers options in the Developer Options of the Mobile Test Plan:
https://docs.google.com/spreadsheets/d/1X9YAf_g1Cq9vZZdCkcBLnRSFrhZuk_vEyCSM3sD5QvM/edit#gid=1556040007
Also added it corresponding .ccz file

## Safety Assurance

- [ ] If the PR is high risk, "High Risk" label is set
- [ ] I have confidence that this PR will not introduce a regression for the reasons below
- [ ] Do we need to enhance manual QA test coverage ? If yes, "QA Note" label is set correctly

### Automated test coverage
This testcase covers the below scenarios:
1. Tests enabling Developers Options
2. Verifies Update Options are enabled
3. Verifies Image Above Question Text is enabled
4. Verifies Image Above Question Text is disabled

### Safety story
Executed in local AVD, ran successfully.

### CODE REVIEW

**Comment by dimagimon** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not contain any JIRA issue keys in the PR title or commit messages (e.g. KEY-123)</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by shubham1g5** [comment]
@damagatchi retest this please

File: app/instrumentation-tests/src/org/commcare/androidTests/DeveloperOptionsTests.kt [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            .perform(click())
+        onView(withText("Image Above Question Text Enabled"))
+            .perform(click())
+        onView(withText("Enabled"))
+            .perform(click())
+        InstrumentationUtility.gotoHome()
+        InstrumentationUtility.openForm(0,0)
+        onView(withId(R.id.image)).check(isAbove(withSubstring("Enter a Name")))
+        onView(withClassName(endsWith("EditText"))).perform(typeText("Image above text"))
+        InstrumentationUtility.submitForm()
```
Comment (shubham1g5) [inline app/instrumentation-tests/src/org/commcare/androidTests/DeveloperOptionsTests.kt:111]: So I think we can simply exit the form without submitting here as that doesn't test anything related to this test. Also the same comment for `testImageBelowText`

**Review by shubham1g5 — CHANGES_REQUESTED** [review]

File: app/instrumentation-tests/src/org/commcare/androidTests/DeveloperOptionsTests.kt (hunk 1 above)
Comment (kbo001) [reply app/instrumentation-tests/src/org/commcare/androidTests/DeveloperOptionsTests.kt]: replaced `submitForm()` calls with `exitForm()`

**Comment by shubham1g5** [comment]
@damagatchi retest this please

**Comment by avazirna** [comment]
@damagatchi retest this please

**Comment by shubham1g5** [comment]
@damagatchi retest this please

**Comment by shubham1g5** [comment]
@damagatchi retest this please

**Comment by shubham1g5** [comment]
@damagatchi retest this please

**Comment by shubham1g5** [comment]
@damagatchi retest this please

**Review by shubham1g5 — APPROVED** [review]

**Pull request merged**